src/_P022_PCA9685.ino

```diff
@@ -184,36 +191,34 @@ boolean Plugin_022(uint8_t function, struct EventStruct *event, String& string)
         break;
       }
       String log;
-      String line           = string;
-      String command;
-      int    dotPos          = line.indexOf('.');
       bool   instanceCommand = false;
+      String command         = parseString(string, 1);
+      int8_t dotPos          = command.indexOf('.');
 
       if (dotPos > -1)
       {
         LoadTaskSettings(event->TaskIndex);
-        String name = line.substring(0, dotPos);
+        String name = command.substring(0, dotPos);
```

Comment:
There should be a check here to see if dotPos is before any command argument separator.

Reply:
`command` is parsed from `string`, a few lines above, so that won't have any argument separators. That is the actual fix 😉

Reply:
Ah you're right :)